plugins/romega/const_omega.ml

```diff
@@ -7,15 +7,14 @@
  *************************************************************************)
 
 open Names
-open Constr
```

Comment:
I think `EConstr` is designed to be open _after_ `Constr`, that's a matter of style of course, but if you chose to do that the diff will become much smaller.

Reply:
I think so as well.